feat(http): Update some mentions of "payload" -> "content" (#35335)
mdn/content: files/en-us/glossary/representation_header/index.md

{{GlossarySidebar}}

> A **representation header** (or 'representation metadata') is an {{glossary("HTTP_header", "HTTP header")}} that describes a particular _representation_ of a resource.
> When a message includes content, the representation header fields describe how to interpret the data.

Reviewer: I've been playing around with this, because it seems odd to talk about this without defining what a representation is. I think I prefer this.

```suggestion
A **representation header** (or 'representation metadata') is an {{glossary("HTTP_header", "HTTP header")}} that describes how to interpret the data contained in the message.
```

Author: This is better 👍🏻

> For example, data might be formatted as a particular media type such as XML, JSON, HTML or Markdown, or localized to a particular written language or geographical region.

Reviewer: ```suggestion
For example, the content in a particular message might be encoded for transport, the whole resource might be formatted as a particular media type such as XML, JSON, HTML or Markdown, compressed using a particular algorithm, or localized to a particular written language or geographical region.
```

Author: Super, looks good tnx
